firstCodeInPython.py

```diff
@@ -0,0 +1,21 @@
+text = open('text.txt', 'r') 
+d = dict() 
+for line in text: 
+	 line = line.strip() 
+	 line = line.lower() 
+	 words = line.split(" ") 
+
+	 for word in words: 
+	 	if word in d:
+	 		d[word] = d[word] + 1
+	 	else: 
+	 		d[word] = 1
+for key in list(d.keys()): 
+    print(key, ":", d[key]) 
+
+mostapperdword=max(d,key=d.get)
```

Comment:
according to python style guide, var name should look like this: most_apperd_word